### ISSUE TITLE
Nesting 2 Subsections breaks customizer when traversing backwards from last subsection

### ISSUE THREAD

**Issue by noknokcody** [themeum/kirki#2386]
### Issue description:
Ran into this one while testing the capabilities of Kirki's subsections. I've found that if you nest a subsection inside another subsection you'll break the customizer when traversing backward.

### GIF Demonstration:

![GIF Demonstration](https://i.imgur.com/j2jhPD0.gif)

### Version used:
3.1.6

### Using theme_mods or options?
theme_mods

### Code to reproduce the issue (config + field(s))
``` PHP

define( 'NOK_KIRKI_CONFIG', 'nok_kirki_config' );

Kirki::add_config( NOK_KIRKI_CONFIG, [
    'capability'  => 'edit_theme_options',
    'option_type' => 'theme_mod'
] );

Kirki::add_panel( 'test_panel', [ 
    'priority'    => 10000,
    'title'       => nok_esc_html__( 'Test Panel' ),
    'description' => nok_esc_html__( 'Testing purposes only' )
]);

Kirki::add_section( 'test_section_1', [
	'title'         => 'Test 1',
	'description'   => nok_esc_html__( '' ),
	'panel'         => 'test_panel'
] );

Kirki::add_field( NOK_KIRKI_CONFIG, [
    'settings'	=> 'test_field_1',
    'label'		=> nok_esc_html__( 'Test Control 1' ),
    'type'		=> 'toggle',
    'section'	=> 'test_section_1',
    'default'	=> 0,
    'priority'  => 20,
] );


Kirki::add_section( 'test_section_2', [
	'title'         => 'Test 2',
	'description'   => nok_esc_html__( '' ),
	'section'         => 'test_section_1'
] );

Kirki::add_field( NOK_KIRKI_CONFIG, [
    'settings'	=> 'test_field_2',
    'label'		=> nok_esc_html__( 'Test Control 2' ),
    'type'		=> 'toggle',
    'section'	=> 'test_section_2',
    'default'	=> 0,
    'priority'  => 20,
] );

Kirki::add_section( 'test_section_3', [
	'title'         => 'Test 3',
	'description'   => nok_esc_html__( '' ),
	'section'         => 'test_section_2'
] );

Kirki::add_field( NOK_KIRKI_CONFIG, [
    'settings'	=> 'test_field_3',
    'label'		=> nok_esc_html__( 'Test Control 3' ),
    'type'		=> 'toggle',
    'section'	=> 'test_section_3',
    'default'	=> 0,
    'priority'  => 20,
] );

```

**Comment by mapsteps** [comment]
Hello and thank you for reporting this issue.

We are closing most open GitHub issues with the release of Kirki 4 to start fresh with this major release.

If you are still running into issues after you have tested your theme with Kirki 4, please let us know so we can reopen the issue.

Please note that Kirki 3 is no longer actively supported. Moving forward, we will only provide support the very latest version of Kirki.

Thank you for your understanding.

Best,
David

**Comment by noknokcody** [comment]
Can confirm this is resolved in Kirki 4.0.22
